src/main/java/org/wildfly/extension/elytron/FilteringKeyStoreService.java

```diff
@@ -57,6 +59,15 @@ public void start(StartContext startContext) throws StartException {
             KeyStore unmodifiable = keyStoreService.getValue();
             KeyStore modifiable = keyStoreService.getModifiableValue();
 
+            if (ROOT_LOGGER.isTraceEnabled()) {
```

Comment:
Use tracef (see comments on https://github.com/wildfly-security/wildfly-elytron/pull/502 for more info).